timely/src/logging.rs

```diff
@@ -70,9 +72,44 @@ pub struct ChannelsEvent {
     pub target: (usize, usize),
 }
 
-#[derive(Serialize, Deserialize, Abomonation, Debug, Clone, Hash, Eq, PartialEq, Ord, PartialOrd)]
+#[cfg(not(feature = "bincode"))]
+/// Encapsulates (de)serialization, Debug, Display, for dynamically typed timestamps in logs
+pub trait ProgressEventTimestamp: std::fmt::Debug + std::fmt::Display + std::any::Any {
+    /// Encodes a typed reference into a binary buffer.
```

Comment:
Just to check, `Timestamp` has already `Debug+Any+Abomonation/Serialize`. Is the motivation for the trait to add `Display`, or because the `communication::Data` trait isn't actually helpful?

Reply:
It’s because Data can’t be turned into a trait, because entomb takes a type parameter. Should probably add a comment to remind future readers.